rmarkdowns/Week_3.Rmd

```diff
@@ -217,8 +278,48 @@ ggplot() +
     x = "Longitude",
     y = "Latitude"
   )
+
+######################
+
+# visualising all retrieved features over each other to form a map of Oahu
+ggplot() +
+  geom_sf(data = streets_oahu$osm_lines, inherit.aes = FALSE, color = "#ffbe7f", size = .4, alpha = .8) +
+  geom_sf(data = small_streets_oahu$osm_lines, inherit.aes = FALSE, color = "#a6a6a6", size = .2, alpha = .8) +
+  geom_sf(data = coastline_oahu$osm_lines, inherit.aes = FALSE, color = "black", size = .8, alpha = .5) +
+  geom_sf(data = hawaiian$osm_polygons, inherit.aes = FALSE, colour = "#0077be", fill = "#0077be", alpha = .8, size = 1) +
+  geom_sf(data = protected$osm_polygons, inherit.aes = FALSE, colour = "pink", fill = "pink", alpha = .5, size = 1) +
+  coord_sf(xlim = c(-158.28058, -157.64863), ylim = c(21.25482, 21.71201), expand = TRUE) +
+  geom_sf_text(data = hawaiian$osm_polygons, aes(label = name), size = 2, hjust = 0) +
+  geom_sf_text(data = protected$osm_polygons, aes(label = name), size = 2, hjust = 0) +
+  theme_bw() +
+  labs(
+    title = "Hawaiian Lands (blue) and protected lands (pink)",
+    x = "Longitude",
+    y = "Latitude"
+  )
+
+# another map, of higher education on Oahu
+ggplot() +
+  geom_sf(data = streets_oahu$osm_lines, inherit.aes = FALSE, color = "#ffbe7f", size = .4, alpha = .8) +
+  geom_sf(data = small_streets_oahu$osm_lines, inherit.aes = FALSE, color = "#a6a6a6", size = .2, alpha = .8) +
+  geom_sf(data = coastline_oahu$osm_lines, inherit.aes = FALSE, color = "black", size = .8, alpha = .5) +
+  geom_sf(data = uni$osm_polygons, inherit.aes = FALSE, colour = "#0077be", fill = "#0077be", alpha = .8, size = 1) +
+  geom_sf(data = cc$osm_polygons, inherit.aes = FALSE, colour = "red", fill = "red", alpha = .5, size = 1) +
+  coord_sf(xlim = c(-158.28058, -157.64863), ylim = c(21.25482, 21.71201), expand = TRUE) +
+  geom_sf_text(data = uni$osm_polygons, aes(label = name), size = 2, hjust = 0) +
+  geom_sf_text(data = cc$osm_polygons, aes(label = name), size = 2, hjust = 0) +
+  theme_bw() +
+  labs(
+    title = "Universities (blue) and Community Colleges (red) on Oʻahu",
```

Comment:
This looks great! Really shows how far some people would have to commute to get to higher ed!